aten/src/ATen/native/tags.yaml

```diff
@@ -63,3 +63,8 @@
           Pointwise operators are operators where each element of the output is computed only by accessing
           the corresponding element of all the broadcasted inputs. The output shape will be the broadcasted
           shape of the inputs.
+- tag: maybe_aliasing_or_mutating
+  desc: |
+          For some ops, we can't statically determine whether the op is functional or not. It is useful to
+          know this information for export as we would want to decompose these ops as they are unsafe to be
+          preserved.
```

Comment:
Maybe worth a comment that this tag should only apply to ops that decompose before functionalization/autograd (CompositeImplicitAutograd). Since autograd/functionalization need to statically know schema information.